h/static/scripts/tags.coffee

```diff
@@ -14,24 +14,24 @@ createTagHelpers = ['localstorage', (localstorage) ->
 
   # Add newly added tags from an annotation to the stored ones and refresh
   # timestamp for every tags used.
-  storeTags: (tags) ->
-    savedTags = localstorage.getObject TAGS_MAP_KEY
+  store: (tags) ->
+    savedTags = localStorage.getObject TAGS_MAP_KEY
```

Comment:
You've changed the format of `savedTags`, but you don't have any code that handles backwards compatibility with the previous format.